internal/pkg/deploy/cloudformation/cloudformation_integration_test.go

```diff
@@ -49,7 +50,7 @@ func Test_App_Infrastructure(t *testing.T) {
 	callerInfo, err := identity.Get()
 	require.NoError(t, err)
 	require.NoError(t, err)
-	deployer := cloudformation.New(sess)
+	deployer := cloudformation.New(sess, cloudformation.WithProgressTracker(os.Stderr))
```

Comment:
ooh why did this fix it? it should have been optional to enable a progress tracker in the first place 🤔